custom_components/vestaboard/config_flow.py

```
@@ -236,3 +258,59 @@ async def _abort_if_configured(
+                        {
+                            CONF_HOST: new_host,
+                            CONF_API_KEY: self.config_entry.data[CONF_API_KEY],
+                        },
+                    )
+                    if await client.check_endpoint() == EndpointStatus.UNKNOWN:
+                        errors[CONF_HOST] = "invalid_host"
+                except asyncio.TimeoutError:
+                    errors[CONF_HOST] = "timeout_connect"
+                except ClientConnectorError:
+                    errors[CONF_HOST] = "invalid_host"
+                except Exception as ex:  # pylint: disable=broad-except
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)